connector/src/main/java/org/geysermc/connector/skin/FloodgateSkinUploader.java

```diff
@@ -132,18 +132,10 @@ public void onMessage(String message) {
                         case LOG_MESSAGE:
                             String logMessage = node.get("message").asText();
                             switch (node.get("priority").asInt()) {
-                                case -1:
-                                    logger.debug("Got a message from skin uploader: " + logMessage);
-                                    break;
-                                case 0:
-                                    logger.info("Got a message from skin uploader: " +logMessage);
-                                    break;
-                                case 1:
-                                    logger.error("Got a message from skin uploader: " + logMessage);
-                                    break;
-                                default:
-                                    logger.info(logMessage);
-                                    break;
+                                case -1 -> logger.debug("Got a message from skin uploader: " + logMessage);
+                                case 0 -> logger.info("Got a message from skin uploader: " + logMessage);
+                                case 1 -> logger.error("Got a message from skin uploader: " + logMessage);
```

Comment:
These can be collated
```suggestion
                                case -1, 0, 1 -> logger.debug("Got a message from skin uploader: " + logMessage);
```